query-node/mappings/src/common.ts

```diff
@@ -41,20 +42,41 @@ export function invalidMetadata(extraInfo: string, data?: unknown): void {
   logger.info(errorMessage, data)
 }
 
+/*
+  Creates a predictable and unique ID for the given content.
+*/
+export async function createPredictableId(db: DatabaseManager): Promise<string> {
```

Comment:
The term predictable id is not the best name, why not simply.

```suggestion
export async function getNextId(db: DatabaseManager): Promise<string> {
```

Reply:
The function has been renamed to `getNextId`.